### ISSUE TITLE
Closed: Correlate Goals with plugins and WordPress configuration

### ISSUE THREAD

**Issue by richtabor** [Automattic/wp-calypso#95491]
As part of broader onboarding work, we want to ensure that whatever goal(s) a user selects, they are met with pre-configured experiences that align with their goal(s). This includes which plugins are set up for them and what WordPress settings make the most sense out of the box for them. For example, someone wanting to start a blog will likely want to see a list of their latest posts on the homepage compared to someone who wants to promote a business and might instead want a page.

As part of this, we need to determine what plugins and WordPress settings makes sense for each goal and that's what this issue is dedicated towards.

**Comment by annezazu** [comment]
Asked each plugin to help identify features for each Goal: 

p1HpG7-uTr-p2
p6q8Tx-5VL-p2
p6rkRX-7lE-p2
pabtAt-2Mw-p2

**Comment by nuriapenya** [comment]
Hello! Isn't this issue a duplication of the work we are doing here? https://github.com/orgs/Automattic/projects/911/views/27
We are tackling the downstream experience for each individual goal in that epic, if I am not mistaken.

cc. @autumnfjeld

**Comment by autumnfjeld** [comment]
@nuriapenya  Yes, this issue is not really needed anymore.  At the time I made it I thought maybe dividing LP and product config made sense.  But that doesn't make sense anymore. 

I called this out in my post: pet6gk-21p-p2

![Image](https://github.com/user-attachments/assets/e28f0fa7-a6cd-4f77-b46f-4c5ab69b2c91)

My plan is to remove this issue and rename the issues in [Goals & Launchpad Tasks #95496](https://github.com/Automattic/wp-calypso/issues/95496)  . 

For example from [Launchpad tasks: Publish a blog #98253](https://github.com/Automattic/wp-calypso/issues/98253). to something like  "Launchpad and configuration: Publish a blog"

Does that make sense?  

I'll try to make these changes in the next few days.

cc: @Automattic/quake  @ouikhuan @andres-blanco

**Comment by autumnfjeld** [comment]
Closed in favor of one single parent issue: [Goals & Launchpad Tasks & Product Configuration](https://github.com/Automattic/wp-calypso/issues/95496#top)
